apps/astarte_trigger_engine/lib/astarte_trigger_engine/cqex_nodes.ex

```diff
@@ -0,0 +1,48 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2020 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.TriggerEngine.CQExNodes do
+  use Skogsra.Type
+
+  @default_port 9042
+
+  @impl Skogsra.Type
+  def cast(value)
+
+  def cast(value) when is_binary(value) do
+    value
+    |> String.split(",", trim: true)
+    |> Enum.map(fn host_port_str ->
+      [host, port_str] = String.split(host_port_str, ":")
```

Comment:
This still does not work if `host_port_str` is, for example, `"localhost"`. This would make it crash here, while it should produce `{"localhost", 9042}`.

Hint:
```elixir
case String.split(host_port_str, ":") do
```
etc